### ISSUE TITLE
orderBy typing error when using array of columns

### ISSUE THREAD

**Issue by naishe** [Vincit/objection.js#1354]
I am having an issue in TypeScript when using order by this way: `.orderBy([{ column: 'isConfigured', order: 'asc' }, { column: orderBy, order: ascDesc }]);` citing:

```
Argument of type '{ column: string; order: string; }[]' is not assignable to parameter of type 'ColumnRef'.
  Type '{ column: string; order: string; }[]' is not assignable to type 'string'.ts(2345)
```

The [Knex.js documentation](https://knexjs.org/#Builder-orderBy) says this should be possible. Am I missing something?

Thanks!

**Comment by koskimas** [comment]
> Am I missing something?

Nope, just a case of missing typings. A PR would get merged immediately.

**Comment by real-artswan** [comment]
Hi, what is the version where it is fixed? It seems 1.6.9 has no this fix.
